### PR TITLE
3 connect app to database

### LINKED ISSUE #3: Setup and connect app to database
Currently there is no database connected to the project. The goal of this task is to setup postgres database and connect to the app. This database should also be set up on the staging app on Heroku.
Also, update the readme on how users can set up the app with this database locally.

### PR DESCRIPTION
**Pull request by seun-beta**
Fixes #3

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 app = Flask(__name__)
 
+a = app.config.from_object(os.environ['APP_SETTINGS'])
+db = SQLAlchemy(app)
+
+
 @app.route("/")
 def hello_world():
-    return "<p> Hello World.! </p>"
+    return "<p> Hello World.! </p>"
```
Comment (kosyfrances) [inline app.py]: Needs new line

File: config.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class StagingConfig(Config):
+    DEBUG = True
+    DEVELOPEMENT = True
+
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    DEVELOPEMENT = True
```
Comment (kosyfrances) [inline config.py]: New line needed?

File: config.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import os 
+
+class Config(object):
```
Comment (kosyfrances) [inline config.py]: you can skip the object argument

**Review by Nkoli — APPROVED** [review]
LGTM

**Comment by seun-beta** [comment]
> LGTM

Thank you very much.

**Pull request merged**